File: src/main/java/com/gimranov/zandy/app/model/Creator.java

package com.gimranov.zandy.app.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

@lombok.Data
public class Creator implements Serializable
{

    @SerializedName("creatorType")
    @Expose
    public String creatorType;
    @SerializedName("name")
    @Expose
    public String name;
    @SerializedName("firstName")
    @Expose
    public String firstName;
    @SerializedName("lastName")
    @Expose
    public String lastName;
    private final static long serialVersionUID = -1651706056337494667L;

}

File: src/main/java/com/gimranov/zandy/app/model/Creator.kt
package com.gimranov.zandy.app.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

data class Creator(
        @SerializedName("creatorType")
        @Expose
        var creatorType: String? = null,
        @SerializedName("name")
        @Expose
        var name: String? = null,
        @SerializedName("firstName")
        @Expose
        var firstName: String? = null,
        @SerializedName("lastName")
        @Expose
        var lastName: String? = null
)
